### ISSUE TITLE
False negatives for Twitter links

### ISSUE THREAD

**Issue by irazasyed** [sindresorhus/awesome-lint#88]
The linter always returns false negatives for any Twitter URL. Any repo it's the same thing. At first I thought it could be my network or something but then there's another report from other dev. and also I tested with another connection and config too. Same case regardless.

Here's another report: https://github.com/sindresorhus/awesome/pull/1590#issuecomment-515800323

And here's the lint report for one of the open PRs:
```
awesome-lint https://github.com/ember-community-russia/awesome-ember
✖ Linting

  README.md:630:3
  ✖   630:3  Link to https://github.com/twokul/ember-cli-pact is dead  remark-lint:no-dead-urls
  ✖  1203:3  Link to https://twitter.com/emberjs is dead               remark-lint:no-dead-urls
  ✖  1204:3  Link to https://twitter.com/embertimes is dead            remark-lint:no-dead-urls
  ✖  1205:3  Link to https://twitter.com/EmberWatch is dead            remark-lint:no-dead-urls
  ✖  1206:3  Link to https://twitter.com/EmberWeekly is dead           remark-lint:no-dead-urls
  ✖  1208:3  Link to https://twitter.com/tomdale is dead               remark-lint:no-dead-urls
  ✖  1209:3  Link to https://twitter.com/wycats is dead                remark-lint:no-dead-urls
  ✖  1210:3  Link to https://twitter.com/melaniersumner is dead        remark-lint:no-dead-urls
  ✖  1211:3  Link to https://twitter.com/jwwweber is dead              remark-lint:no-dead-urls
  ✖  1212:3  Link to https://twitter.com/rwjblue is dead               remark-lint:no-dead-urls
  ✖  1213:3  Link to https://twitter.com/stefanpenner is dead          remark-lint:no-dead-urls
  ✖  1214:3  Link to https://twitter.com/mixonic is dead               remark-lint:no-dead-urls
  ✖  1215:3  Link to https://twitter.com/Runspired is dead             remark-lint:no-dead-urls
  ✖  1216:3  Link to https://twitter.com/pzuraq is dead                remark-lint:no-dead-urls
  ✖  1217:3  Link to https://twitter.com/twokul is dead                remark-lint:no-dead-urls
  ✖  1218:3  Link to https://twitter.com/terzicigor is dead            remark-lint:no-dead-urls
  ✖  1219:3  Link to https://twitter.com/dgeb is dead                  remark-lint:no-dead-urls
  ✖  1221:3  Link to https://twitter.com/alexspeller is dead           remark-lint:no-dead-urls
  ✖  1222:3  Link to https://twitter.com/samselikoff is dead           remark-lint:no-dead-urls
  ✖  1223:3  Link to https://twitter.com/ebryn is dead                 remark-lint:no-dead-urls
  ✖  1224:3  Link to https://twitter.com/gavinjoyce is dead            remark-lint:no-dead-urls
  ✖  1225:3  Link to https://twitter.com/ryantotweets is dead          remark-lint:no-dead-urls
  ✖  1226:3  Link to https://twitter.com/baaz is dead                  remark-lint:no-dead-urls
  ✖  1227:3  Link to https://twitter.com/lukemelia is dead             remark-lint:no-dead-urls

  24 errors
```

All the Twitter URLs are active and valid yet it fails.

**Comment by HorlogeSkynet** [comment]
It looks like rate limit.
Have you try with a developer token or something unlocking bulk requests on those platforms ?

**Comment by irazasyed** [comment]
We can't set a custom token AFAIK and I don't think it relies on the API either. It makes HTTP HEAD requests to verify based on the status code.

**Comment by AdrienTorris** [comment]
I have a false negative too on my PR's awesome-lint report.

**Comment by sindresorhus** [comment]
See https://github.com/transitive-bullshit/check-links/issues/4

**Comment by transitive-bullshit** [comment]
There is definitely a broader issue here with the `remark-lint:no-dead-urls` rule. See my proposal in #89 to work around it.

Even with that proposal, we should still figure out what's going on with Twitter URLs, though transitive-bullshit/check-links#4 is the place to continue that conversation.

Thanks for the report!
